### PR TITLE
added spp_country_office module

### PR DESCRIPTION
**Pull request by reichie020212**
- Added new module named "spp_country_office"
- Added Menu named "Country Office" located in Registry -> Configuration -> Country Office
- Created a new tree and form based on the base tree and form of res.company

### CODE REVIEW

File: spp_country_office/views/country_office_views.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            </form>
+        </field>
+    </record>
+
+    <record id="view_country_office_tree" model="ir.ui.view">
+        <field name="name">view_country_office_tree</field>
+        <field name="model">res.company</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" string="Country Office" />
```
Comment (nhatnm0612) [inline spp_country_office/views/country_office_views.xml]: dunno if a list view with only 1 field is a good idea, @reichie020212

**Review by nhatnm0612 — APPROVED** [review]

File: spp_country_office/views/country_office_views.xml (hunk 1 above)
Comment (reichie020212) [reply spp_country_office/views/country_office_views.xml]: Added a new field

**Review by gonzalesedwin1123 — APPROVED** [review]

**Comment by gonzalesedwin1123** [comment]
@reichie020212 please fix the automated test error

**Comment by gonzalesedwin1123** [comment]
@reichie020212 please fix the automated test

**Pull request merged**